packages/ra-ui-materialui/src/input/TimeInput.tsx

```diff
@@ -87,6 +91,7 @@ export const TimeInput = ({
             size="small"
             variant={variant}
             margin={margin}
+            disabled={disabled || readOnly}
```

Comment:
```suggestion
            disabled={disabled || readOnly}
            readOnly={readOnly}
```

To be consistent with `DateInput` and `DateTimeInput`